### DISCUSSION TITLE
Read only the last sheet

### DISCUSSION THREAD

**Discussion by DarkTama** [ozlerhakan/poiji#271] [Q&A]
I'd like to read the last sheet added that will probably be on the furthest right in the excel.

**Comment by DarkTama** [comment]
`import com.poiji.bind.Poiji;
import com.poiji.exception.PoijiExcelType;
import com.poiji.option.PoijiOptions;

import java.io.File;
import java.util.List;

public class ExcelReader {

    public static void main(String[] args) {
        File file = new File("example.xlsx");

        // set options for the Poiji library
        PoijiOptions options = PoijiOptions.PoijiOptionsBuilder.settings().build();
        
        // get the list of sheets in the Excel file
        List<String> sheetNames = Poiji.getSheetNames(file, PoijiExcelType.XLSX);
        
        // get the last sheet name
        String lastSheetName = sheetNames.get(sheetNames.size() - 1);
        
        // read only the last sheet
        List<Object> sheetData = Poiji.fromExcel(file, lastSheetName, PoijiExcelType.XLSX, options);
        
        // process the sheet data here
        for (Object row : sheetData) {
            // process row data here
            System.out.println(row.toString());
        }
    }
}`

got this from chatgpt

**Reply by ozlerhakan** [reply]
Hi @DarkTama , 

I'm guessing that we don't have such method `List<String> sheetNames = Poiji.getSheetNames(file, PoijiExcelType.XLSX);` in Poiji :D
